source/OrdinalNotions.lagda

```diff
@@ -300,12 +304,100 @@ the time of writing, namely 11th January 2021).
 
 \begin{code}
 
+in-trichotomy : (x y : X) → 𝓤 ⊔ 𝓥 ̇
+in-trichotomy x y = (x < y) + (x ≡ y) + (y < x)
+
 is-trichotomous-element : X → 𝓤 ⊔ 𝓥 ̇
-is-trichotomous-element x = (y : X) → (x < y) + (x ≡ y) + (y < x)
+is-trichotomous-element x = (y : X) → in-trichotomy x y
 
 is-trichotomous-order : 𝓤 ⊔ 𝓥 ̇
 is-trichotomous-order = (x : X) → is-trichotomous-element x
 
+-- injections into in-trichotomy
+>-implies-in-trichotomy : {x y : X} → (x < y) → in-trichotomy x y
+>-implies-in-trichotomy = inl
+
+≡-implies-in-trichotomy : {x y : X} → (x ≡ y) → in-trichotomy x y
+≡-implies-in-trichotomy = inr ∘ inl
+
+<-implies-in-trichotomy : {x y : X} → (y < x) → in-trichotomy x y
+<-implies-in-trichotomy = inr ∘ inr
+
+in-trichotomy-symm : {x y : X} → in-trichotomy x y → in-trichotomy y x
+in-trichotomy-symm (inl x-lt-y) = inr (inr x-lt-y)
+in-trichotomy-symm (inr (inl x-equiv-y)) = inr (inl (x-equiv-y ⁻¹))
+in-trichotomy-symm (inr (inr y-lt-x)) = inl y-lt-x
+
+[_,_] : ∀ {𝓠 𝓡 𝓦} {X : 𝓠 ̇} {Y : 𝓡 ̇} {Z : 𝓦 ̇} → (X → Z) → (Y → Z) → (X + Y → Z)
+[ f , g ] (inl x) = f x
+[ f , g ] (inr x) = g x
+
+_>_ : (x y : X) → 𝓥 ̇
+x > y = y < x
+
+_≦_ : (x y : X) → 𝓤 ⊔ 𝓥 ̇
+x ≦ y = (x < y) + (y ≡ x)
+
+_≧_ : (x y : X) → 𝓤 ⊔ 𝓥 ̇
+x ≧ y = (x ≡ y) + (y < x)
+
+coprod-symm : ∀ {𝓠 𝓡} {X : 𝓠 ̇} {Y : 𝓡 ̇} → X + Y → Y + X
```

Comment:
I do need to introduce them if I want to use this function in `≧-implies-≦` and `≦-implies-≧`, since in one of them `{X : 𝓤 ̇}` while in the other `{X :  𝓥 ̇}`. But I will use U, V, W intstead!

Reply:
In any case, used the predefined one from `Plus-Properties` so this is now resolved.